llvm/lib/Transforms/Utils/Local.cpp

```diff
@@ -3648,6 +3648,33 @@ DIExpression *llvm::getExpressionForConstant(DIBuilder &DIB, const Constant &C,
   return nullptr;
 }
 
+void llvm::remapDebugVariable(ValueToValueMapTy &Mapping, Instruction *Inst) {
+  auto RemapDebugOperands = [&Mapping](auto *DV, auto Set) {
+    for (auto *Op : Set)
+      if (Instruction *InstOp = dyn_cast<Instruction>(Op)) {
```

Comment:
I don't think that we need the `dyn_cast` checks now that you're passing in a `ValueToValueMapTy` given that the key is a `Value *`. Could you remove them please?